CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - [#202](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/202) The method #get_default_settings in LogicalInterconnectGroup is used on integration test
 - [#125](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/125) References to resources C7000 in Synergy integration tests
 - [#212](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/212) Unable to create a Server Profile with Deployment Plan settings
+- [#89](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/89) Fix like? method for Logical Interconnect Groups
```

Comment:
@jsmartt , because we will start api500 development soon. After these PRs are merged into the master, I will open a PR of that code in the private github to develop the API500. Let me know your thoughts on this.